unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -168,29 +188,122 @@ function cert_generateRootCAcertificate() {
 
 }
 
-function cert_parseYaml() {
+function cert_parseYaml {
```

Comment:
GJ, could you add the result of the new function when evaluating a config.yml?

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1770/commits/4ea226db5e29ede078f86b3b799eb04b1a5bac84